pkg/monitoring/rules/rules.go

```diff
@@ -1,180 +1,56 @@
-				healthImpactAlertLabelKey: "critical",
-				partOfAlertLabelKey:       partOfAlertLabelValue,
-				componentAlertLabelKey:    componentAlertLabelValue,
-			},
-		},
-		{
-			Alert: "SSPFailingToReconcile",
-			Expr:  intstr.FromString("(kubevirt_ssp_operator_reconcile_succeeded_aggregated == 0) and (kubevirt_ssp_operator_up > 0)"),
-			For:   ptr.To[promv1.Duration]("5m"),
-			Annotations: map[string]string{
-				"summary":     "The ssp-operator pod is up but failing to reconcile",
-				"runbook_url": fmt.Sprintf(runbookURLTemplate, "SSPFailingToReconcile"),
-			},
-			Labels: map[string]string{
-				severityAlertLabelKey:     "critical",
-				healthImpactAlertLabelKey: "critical",
-				partOfAlertLabelKey:       partOfAlertLabelValue,
-				componentAlertLabelKey:    componentAlertLabelValue,
-			},
-		},
-		{
-			Alert: "SSPHighRateRejectedVms",
-			Expr:  intstr.FromString("kubevirt_ssp_template_validator_rejected_increase > 5"),
-			For:   ptr.To[promv1.Duration]("5m"),
-			Annotations: map[string]string{
-				"summary":     "High rate of rejected Vms",
-				"runbook_url": fmt.Sprintf(runbookURLTemplate, "SSPHighRateRejectedVms"),
-			},
-			Labels: map[string]string{
-				severityAlertLabelKey:     "warning",
-				healthImpactAlertLabelKey: "warning",
-				partOfAlertLabelKey:       partOfAlertLabelValue,
-				componentAlertLabelKey:    componentAlertLabelValue,
-			},
-		},
-		{
-			Alert: "SSPCommonTemplatesModificationReverted",
-			Expr:  intstr.FromString("kubevirt_ssp_common_templates_restored_increase > 0"),
-			For:   ptr.To[promv1.Duration]("0m"),
-			Annotations: map[string]string{
-				"summary":     "Common Templates manual modifications were reverted by the operator",
-				"runbook_url": fmt.Sprintf(runbookURLTemplate, "SSPCommonTemplatesModificationReverted"),
-			},
-			Labels: map[string]string{
-				severityAlertLabelKey:     "warning",
-				healthImpactAlertLabelKey: "none",
-				partOfAlertLabelKey:       partOfAlertLabelValue,
-				componentAlertLabelKey:    componentAlertLabelValue,
-			},
-		},
-		{
-			Alert: "VirtualMachineCRCErrors",
-			Expr:  intstr.FromString("(count(kubevirt_ssp_vm_rbd_block_volume_without_rxbounce > 0) or vector(0)) > 0"),
-			Annotations: map[string]string{
-				"description": "{{ $value }} Virtual Machines are in risk of causing CRC errors and major service outages",
-				"summary":     "When running VMs using ODF storage with 'rbd' mounter or 'rbd.csi.ceph.com provisioner', it will report bad crc/signature errors and cluster performance will be severely degraded if krbd:rxbounce is not set.",
-				"runbook_url": fmt.Sprintf(runbookURLTemplate, "VirtualMachineCRCErrors"),
-			},
-			Labels: map[string]string{
-				severityAlertLabelKey:     "warning",
-				healthImpactAlertLabelKey: "none",
-				partOfAlertLabelKey:       partOfAlertLabelValue,
-				componentAlertLabelKey:    componentAlertLabelValue,
-			},
-		},
-	}
+func ListAlerts() []promv1.Rule {
```

Comment:
This function is not used anywhere, can you remove it? If it will be needed in the future, we can add it.

Reply:
Sure? It can be used and all operator have it, since we use it for recording rules and metrics.. so I want users to know they can list also alerts

Reply:
It would be simple to add it again, when needed. This package is not used by any other code, only ssp-opeartor.

Reply:
Yes but users might don't know they can add it. But If you order I will remove.

Reply:
To give you more explanation why: 
This repository, `ssp-operator`, is an application, not a library. So we can assume that there are no other repositories that import this repository. We, the developers of this project, are the only users of this function.
Our only external API that has to be kept compatible is in the `api` folder. Everything else can be changed when we want to.

It would be different if this project was a library that is imported by others, and we need to provide functions that they can use.

Reply:
Thank you for explaining it!
But personally I don't think we are the only users- it's upstream, many people can contribute to this project or learn from it.
anyway it's removed (: